[CAM-8384] - Document CAM CPC on vCF behavior (#2597)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Deploying Anywhere/using-private-cloud-on-vmware-cloud-foundation.md

{{{
"title": "Using CenturyLink Private Cloud on VMware Cloud Foundation",
"date": "01-23-2019",
"author": "Julio Castanar",

> "keywords": ["cam","centurylink private cloud", "vmware cloud foundation", "deploy", "resource names"],

Reviewer: ```suggestion
"keywords": ["cam","centurylink private cloud", "cpc on vcf", "vmware cloud foundation", "deploy", "provider"],
```

#### Compute

| Option | Description |
|--------|-------------|
| Virtual CPUs | Specify the number of CPUs. |
| Memory | Specify the virtual RAM in GB. |

See the [VMware Cloud Foundation](https://docs.vmware.com/en/VMware-Cloud-Foundation/3.5/com.vmware.vcf.ovdeploy.doc_35/GUID-F2DCF1B2-4EF6-444E-80BA-8F529A6D0725.html) documentation.

#### Disks

| Option | Description |
|--------|-------------|
| Disk | Specify the storage capacity in GB. Adding disk storage capacity is supported. |

See the [VMware Cloud Foundation](https://docs.vmware.com/en/VMware-Cloud-Foundation/3.5/com.vmware.vcf.ovdeploy.doc_35/GUID-F2DCF1B2-4EF6-444E-80BA-8F529A6D0725.html) documentation.

### Naming vCloud resources

When creating an instance, an automatic name is generated and used for VApps, VMs and Hostname, all with the same generated name. This way you can identify unregistered instances.

> This name generated has maximum length of 15 characters (max allowed in windows hostnames), and it's composed by this values:

Reviewer: ```suggestion
The generated name has a maximum length of 15 characters (max allowed in windows hostnames), and it's composed by these values:
```

```
InstanceName-ServiceId-InstanceNumber
```

> In the Policy Box it's possible specify you want more that one instance. An example is “jdedepl-i8oq0-1”.  

Reviewer: ```suggestion
In the Policy Box it is possible to specify more than one machine to be deployed, so the instanceNumber is added as a suffix in the name. An example is “jdedepl-i8oq0-1”.  
```

> Once created this name, **modifying the vApp name is not supported**, because it could affect to the instance operations. However, it's possible to change manually this name on VMs and Hostnames.  

Reviewer: ```suggestion
Once deployed, **modifying the vApp name is not supported**, because it could affect the instance operations through Cloud Application Manager. However, it's possible to change this name manually on VMs and Hostnames.  
```

> In case want to rename a VM/Hostname, it would be needed to do the following steps:

Reviewer: ```suggestion
In case you want to rename a VM/Hostname, you need to perform the following steps:
```
